maintainers/maintainer-list.nix

```diff
@@ -1036,6 +1036,16 @@
     githubId = 75235;
     name = "Michael Walker";
   };
+  bartuka = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add bartuka``.

Reply:
Thanks for feedbacks @SuperSandro2000 , I have this other PR opened https://github.com/NixOS/nixpkgs/pull/117908 where I add me as maintainer, would be better to fix this too and solve the conflict later or remove this code and merge this after #117908 ?